src/entities/Proposal/routes.ts

```diff
@@ -188,22 +189,15 @@ export async function createProposalBanName(req: WithAuth) {
 }
 
 const newProposalPOIValidator = schema.compile(newProposalPOIScheme)
-export async function createProposalPOI(req: WithAuth) {
+type VerifyFunction = (config: NewProposalPOI) => Promise<void>;
```

Comment:
If we change `NewProposalPoI` to have a `poiType: PoiType`, then we could use just one endpoint and validate accordingly. (and also determine how to create the proposal template)